### ISSUE TITLE
Custom Fields removed from FeatureFlagConfigurationSetting

### ISSUE THREAD

**Issue by mrm9084** [Azure/azure-sdk-for-python#34545]
- **Package Name**: azure-appconfiguration
- **Package Version**: 1.5.0
- **Operating System**: Windows 11
- **Python Version**: 3.11

**Describe the bug**
When I make get/list configuration request to Azrue App Configuration any "custom" fields in FeatureFlagConfigurationSetting are ignored/removed.

**To Reproduce**
1. Create a Feature Flag in the Azure App Configuration Portal with the following values:
```json
{
    "id": "Beta",
    "description": "Matt's variant FF",
    "enabled": true,
    "variants": [
        {
            "name": "Off",
            "configuration_value": "Off",
            "status_override": "Enabled"
        },
        {
            "name": "On",
            "configuration_value": "On",
            "status_override": "Disabled"
        }
    ],
    "allocation": {
        "percentile": [
            {
                "variant": "Off",
                "from": 0,
                "to": 100
            }
        ],
        "user": [
            {
                "variant": "Off",
                "users": [
                    "Adam"
                ]
            }
        ],
        "seed": "adfsasdfzsd",
        "default_when_enabled": "Off",
        "default_when_disabled": "Off"
    }
}
```
1. Load your configurations using `beta = client.get_configuration_setting(key=".appconfig.featureflag/Beta")`

If you look in the value of the feature flag all of the custom fields are missing.

**Expected behavior**
The SDK shouldn't remove customer data. The DotNet and Java SDKs support this.

**NOTE**: You are also removing some known fields like description.

Link to the currently GAed Schema: https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v1.1.0.schema.json

**Comment by YalinLi0312** [comment]
After investigating, I found the root cause is that we miss understood what the property "value" should be, we were assuming the value should always be this json schema which caused custom fields missing in our SDK result:
 `{"id": ..., "enabled": ..., "conditions": {"client_filters": [...]}}`

Thanks for reporting the issue!

**Comment by YalinLi0312** [comment]
Hi @mrm9084 , we've fixed the issue in version [1.6.0b2 ](https://pypi.org/project/azure-appconfiguration/1.6.0b2/). Please try on it, thanks!

**Comment by mrm9084** [comment]
@YalinLi0312, just validated seems good.
